pkg/constants/api_name.go

```diff
@@ -4,43 +4,45 @@ type APIName string
 const (
-	PS_GetPriceWithAddress   APIName = "v1/pricestore/chain/:chain/:address"
-	PS_GetTokensPrice        APIName = "v1/pricestore/chain/:chain/tokens"
-	PS_GetLpTokenPrice       APIName = "v1/pricestore/chain/:chain/lptokens"
-	PS_GetLosers             APIName = "v1/pricestore/chain/:chain/losers"
-	PS_GetGainers            APIName = "v1/pricestore/chain/:chain/gainers"
-	PS_GetPriceBySymbol      APIName = "v1/pricestore/:symbol"
-	TS_GetDetailsByContract  APIName = "v1/tokenstore/token/address/:address"
-	TS_GetTokenList          APIName = "v1/tokenstore/token/all"
-	TS_GetTokenBySymbol      APIName = "v1/tokenstore/token/symbol/:symbol"
-	ASSETS_GetTokenAssets    APIName = "v1/:chain/address/:address/assets"
-	ASSETS_GetProfitsAndLoss APIName = "v2/:chain/address/:address/userData"
-	NFT_GetNFTAssets         APIName = "v1/:chain/address/:address/nft-assets"
-	NFT_GetTxns              APIName = "v1/:chain/address/:address/nft-transactions"
-	NFT_GetDetailsByID       APIName = "v1/:chain/address/:address/details"
-	NFT_GetHoldersByID       APIName = "v1/:chain/address/:address/nftholders"
-	TXN_GetTokenTxns         APIName = "v1/:chain/address/:address/transactions"
-	TXN_GetTxnDetails        APIName = "v1/:chain/transactions/:txnID"
-	TXN_GetTokenTxnsV2       APIName = "v2/:chain/address/:address/transactions"
-	PROTO_GetPositions       APIName = "v2/protocols/:protocol/address/:address/positions"
-	PROTO_GetPairs           APIName = "v2/protocols/:protocol/pairs"
+	PS_GetPriceWithAddress       APIName = "v1/pricestore/chain/:chain/:address"
+	PS_GetTokensPrice            APIName = "v1/pricestore/chain/:chain/tokens"
+	PS_GetLpTokenPrice           APIName = "v1/pricestore/chain/:chain/lptokens"
+	PS_GetLosers                 APIName = "v1/pricestore/chain/:chain/losers"
+	PS_GetGainers                APIName = "v1/pricestore/chain/:chain/gainers"
+	PS_GetPriceBySymbol          APIName = "v1/pricestore/:symbol"
+	TS_GetDetailsByContract      APIName = "v1/tokenstore/token/address/:address"
+	TS_GetTokenList              APIName = "v1/tokenstore/token/all"
+	TS_GetTokenBySymbol          APIName = "v1/tokenstore/token/symbol/:symbol"
+	ASSETS_GetTokenAssets        APIName = "v1/:chain/address/:address/assets"
+	ASSETS_GetProfitsAndLoss     APIName = "v2/:chain/address/:address/userData"
+	NFT_GetNFTAssets             APIName = "v1/:chain/address/:address/nft-assets"
+	NFT_GetTxns                  APIName = "v1/:chain/address/:address/nft-transactions"
+	NFT_GetDetailsByID           APIName = "v1/:chain/address/:address/details"
+	NFT_GetHoldersByID           APIName = "v1/:chain/address/:address/nftholders"
+	TXN_GetTokenTxns             APIName = "v1/:chain/address/:address/transactions"
+	TXN_GetTxnDetails            APIName = "v1/:chain/transactions/:txnID"
+	TXN_GetTokenTxnsV2           APIName = "v2/:chain/address/:address/transactions"
+	TXN_GetRawTransactionDetails APIName = "v3/:chain/address/:address/transactions"
```

Comment:
What if someone wants to use this API and not get the transactions in `raw` format

Reply:
Since the standard return is entirely different from the `raw` return, it won't be feasible or useful to write a general function that will end up returning something of type`interface{}`. To resolve this, I will add another API to get a standard response back